core/services/runs.go

```diff
@@ -240,6 +240,22 @@ func ResumePendingTask(
 	return updateAndTrigger(run, store)
 }
 
+func prepareAdapter(
+	taskRun *models.TaskRun,
```

Comment:
If the method isn't used, shall we ✂️ it?

Reply:
It's still used in QueueSleepingTask, and that's why it does not regress.